doc/source/examples/rpt/photon-count-calculation-in-a-cylindrical-vessel/photon-count-calculation-in-a-cylindrical-vessel.rst

```diff
@@ -0,0 +1,325 @@
+    # Detector parameters
+    #---------------------------------------------------
+    subsection detector parameters
+        set detector positions file         = positions.detector
+        set radius                          = 0.0381
+        set length                          = 0.0762
+        set dead time                       = 1e-5
+        set activity                        = 2e6
+        set attenuation coefficient reactor = 10
+    end
+
+.. note::
+    The parameters ``dead time``, ``activity`` and ``attenuation coefficient reactor`` are obtained using the blackbox optimization software `NOMAD <https://www.gerad.ca/en/software/nomad/>`_ . The second example `Tuning Parameters with NOMAD <../tuning-parameters-with-nomad/tuning-parameters-with-nomad.html>`_ explains how we can obtain the values of these parameters using NOMAD.
+	
+
+:raw-html:`<br />`
+
+Running the simulation
+----------------------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``rpt_3d`` executable is within your path, the simulation can be launched by typing:
+
+.. code-block:: text
+
+    rpt_3d rpt_count_calculation.prm
+  
+Lethe will generate a ``.csv`` file with the name specified next to the ``counts file`` parameter in ``rpt_count_calculation.prm``. The generated ``.csv`` file will contain the :math:`(x,y,z)` coordinates of the particle with its respective photon count measured by a given detector. Each detector is identified by its id number (``detector_id``). In this example, as we have only one detector, all values in the ``detector_id`` column should be :math:`0`.
+
+
+.. warning::
+    When running the code with different particle position files, don't forget to change the name of the exporting ``counts file`` in ``rpt_count_calculation.prm`` so that the previous ``.csv`` file isn't overwritten.
+
+
+:raw-html:`<br />`
+
+Results
+--------
+
+To visualize the data and obtain the figures shown below, a python script (``rpt_count_calculation_plot.py``) is provided. When running the script, the name of the ``.csv`` file you wish to open and read must be specified as an argument.
+
+.. tip::
+    You may use the ``rpt_count_calculation_plot.py`` script to plot any other set of data saved in a ``.csv`` file format.
+
+
+Scenario 1: Horizontal translation of a particle along the x-axis 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. image:: images/result_1.png
+    :alt: Scenario 1 results
+    :align: center
+    :name: Results of the 1st scenario
+
+
+In the figure shown above, as one would expect, as the particle approaches the detector, the photon count grows. Such evolution may be explained by the efficiency of the detector getting greater as the particle advances toward the detector's exposed surface. Since the photon's path length in the vessel decreases, :math:`f_a(\alpha, \theta)` increases, and therefore the efficiency gets greater.
```

Comment:
We know that this increase is not only because of decreasing the length of the traveled path inside the reactor but also it relates to the solid angel as well.

Reply:
The solid angel is my new friend.